### PR TITLE
OSDOCS-6605: [ROSA] Update HIPAA text in compliance section

### PR DESCRIPTION
**Pull request by smunje1**
[OSDOCS-6605](https://issues.redhat.com/browse/OSDOCS-6605): [ROSA] Update HIPAA text in compliance section

Aligned team: Service Delivery
OCP version for cherry-picking: enterprise-4.13+
Preview pages: [Click to see the build preview in your browser](https://63100--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security#rosa-policy-compliance_rosa-policy-process-security)
SME review **completed**: @0kashi
QE review **completed**: @xueli181114 
Peer review **completed**: @jab-rh

### CODE REVIEW

**Comment by smunje1** [comment]
Hello @0kashi
Could you help with the SME review for this Jira? Thank you!

**Comment by karanthshashank** [comment]
Hi @0kashi could you please help review this PR to get the Doc updated and released immediately in the current sprint.

File: modules/rosa-policy-security-regulation-compliance.adoc [hunk 1]
```diff
@@ -54,9 +54,9 @@ Any issues that may be discovered are prioritized based on severity. Any issues
 .Security and control certifications for {product-title}
 [cols= "3,3,3",options="header"]
 |===
-| Certification | {product-title} | {hcp-title-first}
+| Compliance | {product-title} | {hcp-title-first}
```
Comment (0kashi) [inline modules/rosa-policy-security-regulation-compliance.adoc:57]: there is a discrepancy in how the product titles are rendering.  On the left it is just "Red Hat OpenShift Service on AWS". On the right it has the acronym.  Do we want to make sure they are similar?

**Comment by 0kashi** [comment]
made a comment relating to the column titles have a slight discrepancy.  Other than that, LGTM.

File: modules/rosa-policy-security-regulation-compliance.adoc (hunk 1 above)
Comment (smunje1) [reply modules/rosa-policy-security-regulation-compliance.adoc:57]: Updated the second column as, "Red Hat OpenShift Service on AWS (ROSA)". This way the second and third column product titles are consistent.

**Comment by smunje1** [comment]
Hello @0kashi 
Thanks for the SME review. I have incorporated your suggestion. Could you please approve if there are no more comments?

**Comment by 0kashi** [comment]
lgtm

**Comment by jab-rh** [comment]
@smunje1, LGTM, thanks!

**Review by skrthomas — COMMENTED** [review]
Merge review LGTM

**Pull request merged**